### PR TITLE
Minimal documentation changes

### PR DESCRIPTION
**Pull request by wtrocki**
fix for #123

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+## RHOAS App Services SDK
+
+### Management SDK's documentation
```
Comment (craicoverflow) [inline docs/README.md:3]:
```suggestion
### Management SDKs documentation
```

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+- [Kafka Management](github.com/redhat-developer/app-services-sdk-go/kafkamgmt/apiv1)
+- [Service Registry Management](github.com/redhat-developer/app-services-sdk-go/registrymgmt/apiv1)
+- [Service Accounts Management](github.com/redhat-developer/app-services-sdk-go/connectorgmt/apiv1)
+
+###  Service SDK's
```
Comment (craicoverflow) [inline docs/README.md:9]:
```suggestion
###  Service SDKs
```

File: docs/apilevels.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+## API stability
+
+SDKs uses following lables
```
Comment (craicoverflow) [inline docs/apilevels.md:3]:
```suggestion
SDKs uses following labels 
```

File: docs/authentication.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
+## Authentication for Management SDK's
```
Comment (craicoverflow) [inline docs/authentication.md:1]:
```suggestion
## Authentication for Management SDKs
```

File: docs/authentication.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+  "ssl-required": "all",
+  "resource": "cloud-services",
+  "public-client": true,
+  "confidential-port": 0
+}
+```
+
+Valid redirect urls: 
+
+`.openshift.com/*, https://prod.foo.redhat.com/*, https://prod.food.redhat.com:1337/*`
```
Comment (craicoverflow) [inline docs/authentication.md:23]: That is applicable to web UIs - will it work for ANY client ID also?

File: docs/authentication.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+For more information about how to use Keycloak.js please refer to [official documentation](https://github.com/keycloak/keycloak-documentation/blob/master/securing_apps/topics/oidc/javascript-adapter.adoc)
```
Comment (craicoverflow) [inline docs/authentication.md:25]: Seems irrelevant here.

File: docs/authentication.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+```bash 
+curl --location --request POST 'https://identity.api.openshift.com/auth/realms/rhoas/protocol/openid-connect/token' \
+--header 'Content-Type: application/x-www-form-urlencoded' \
+--data-urlencode 'client_id=YOURSERVICEACCOUNT' \
+--data-urlencode 'client_secret=YOURSERVICEACCOUNT_SECRET' \
+--data-urlencode 'scope=email' \
+--data-urlencode 'grant_type=client_credentials'
+```
+
+ > NOTE: We are working on SDK for Authentication that will provide out of the box support for connections
```
Comment (craicoverflow) [inline docs/authentication.md:47]: Repeated.

File: docs/authentication.md (hunk 5 above)
Comment (wtrocki) [reply docs/authentication.md:23]: That is the client that we provide. Do you mean adding cli client instead?

Comment (wtrocki) [reply docs/authentication.md:23]: This client will not be useful for golang we should use cli client. Do you mind providing/editing this file with cli client info

File: docs/authentication.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+All Management SDK's are working with Red Hat Single Sign On (sso.redhat.com) server.
+Authentication to services can be done using Keycloak.js library.
+
+When authenticating please use following keycloak client configuration:
+```
+{
+  "realm": "redhat-external",
+  "auth-server-url": "https://sso.redhat.com/auth/",
+  "ssl-required": "all",
+  "resource": "cloud-services",
```
Comment (craicoverflow) [inline docs/authentication.md:15]: Is `resource` client ID? Because this is different for every client consuming the APIs - for example, the KAS-Ui uses `strimzi-ui`, the CLI uses `rhoas-cli-prod` etc.

Comment (wtrocki) [reply docs/authentication.md:15]: Yes.. This is the weakest point right now. 
We might:

- add cli as example client
- remove this chapter

Comment (craicoverflow) [reply docs/authentication.md:15]: I believe we should use the CLI as an example to provide some basis, or just use a placeholder like "<your-client-id>"

**Comment by wtrocki** [comment]
@craicoverflow Thank you so much for the fix
